docs/api/node-events/browser-launch-api.mdx

```diff
@@ -111,14 +109,19 @@ on('before:browser:launch', (browser = {}, launchOptions) => {
 
 :::
 
-#### Add browser extensions:
+### Add browser extensions
+
+**Limitations:**
+
+- Headless Chrome does not support loading extensions.
+- Chrome-branded browsers (e.g., standard Chrome) version 137 and above no longer support extension loading via this API, due to the removal of the --load-extension flag by Chrome. We recommend using Chrome for Testing or Chromium instead. See Cypress Docker image examples for [Chrome for Testing](https://github.com/cypress-io/cypress-docker-images/tree/master/examples/chrome-for-testing?rgh-link-date=2025-05-13T19%3A08%3A57.000Z) and [Chromium](https://github.com/cypress-io/cypress-docker-images/tree/master/examples/chromium?rgh-link-date=2025-05-13T19%3A08%3A57.000Z).
```

Comment:
```suggestion
- Chrome-branded browsers (e.g., standard Chrome) version 137 and above no longer support extension loading via this API, due to the removal of the `--load-extension` flag by Chrome. We recommend using Chrome for Testing or Chromium instead. See Cypress Docker image examples for [Chrome for Testing](https://github.com/cypress-io/cypress-docker-images/tree/master/examples/chrome-for-testing) and [Chromium](https://github.com/cypress-io/cypress-docker-images/tree/master/examples/chromium).
```

If the link dates were in there for a reason, then please ignore! They just looked out of place to me.